code/modules/mob/new_player/preferences_setup.dm

```diff
@@ -138,6 +138,32 @@
 			clothes_s.Blend(new /icon('icons/mob/back.dmi', "satchel"), ICON_OVERLAY)
 	return clothes_s
 
+/datum/preferences/proc/valid_hairstyles(var/gender_restricted = TRUE)
```

Comment:
Because whoever coded this before decided there is no conceivable reason for female characters to grow a beard